src/main/asciidoc/reference/elasticsearch-misc.adoc

```diff
@@ -275,3 +275,42 @@ SearchHits<SomethingToBuy> searchHits = operations.search(query, SomethingToBuy.
 ====
 
 This works with every implementation of the `Query` interface.
+
+[[elasticsearch.misc.point-in-time]]
+== Point In Time (PIT) API
+
+`ElasticsearchOperations` supports the point in time API of Elasticsearch (see https://www.elastic
+.co/guide/en/elasticsearch/reference/8.3/point-in-time-api.html). The following code snippet shows how to use this
```

Comment:
wrong line break here 😄 

[preview](https://docs.spring.io/spring-data/elasticsearch/docs/5.0.0-RC2/reference/html/#elasticsearch.misc.point-in-time)

Reply:
Thanks, I'm currently checking the documentation anyway for the next release, so this will go in there as well.

Reply:
by the way, will PIT be added into 4.x release? I didn't find milestones related to this pr.

Reply:
This change was in 5.0.M6 (I obviously was neglecting setting the milestones when closing issues this summer). So it will be in version 5.

It won't be backported to 4.4. After a version (minor) is released, it will be only updated with bugfixes and dependency patch updates, not with new features or API changes.

Reply:
gotcha, thanks~